Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a Group of VMs/Performance Analysis for a Group of VMs.workbook

```diff
@@ -100,23 +100,23 @@
             "resourceType": "microsoft.resourcegraph/resources"
           },
           {
-            "id": "9732eff8-fb57-4cbd-8ade-5ae746f33760",
+            "id": "2bc1e5fc-cc2d-4eb5-bead-5f7d96664dec",
             "version": "KqlParameterItem/1.0",
             "name": "Workspaces",
             "type": 5,
             "isRequired": true,
             "multiSelect": true,
             "quote": "'",
             "delimiter": ",",
-            "query": "where type =~ 'microsoft.operationalinsights/workspaces'\r\n| summarize by id, name\r\n| project id, selected = iff(id =~ todynamic('{Selection}').ws, true, false)",
+            "query": "// {DefaultWorkspace} {ContextSelection} {Subscriptions}\r\nwhere type =~ 'microsoft.operationalinsights/workspaces'\r\n| summarize by id, name\r\n| order by tolower(name) asc\r\n| extend Row = row_number()\r\n| project id, selected = iff({HybridMode} == 'true', iff(id == todynamic('{ContextSelection}').ws, true, false), Row == 1)",
             "crossComponentResources": [
               "{Subscriptions}"
             ],
             "typeSettings": {
-              "resourceTypeFilter": {
-                "microsoft.operationalinsights/workspaces": true
-              },
-              "additionalResourceOptions": []
+              "limitSelectTo": 5,
+              "additionalResourceOptions": [
+                "value::1"
+              ]
```

Comment:
Can you not change this file? This seems to break in context-free mode (aka Azure Monitor > Workbooks > Virtual Machines > Performance)
![image](https://user-images.githubusercontent.com/43890980/75587912-60f5be00-5a2c-11ea-85d3-3f47d2b3fd70.png)

Reply:
Reverted this file